### PR TITLE
Implement sms sender to send sms to external url

### PR DESCRIPTION
**Pull request by nazaretyan**

### CODE REVIEW

File: infra/infra-integrations/infra-integrations-sms/src/main/java/com/sflpro/notifier/externalclients/sms/webhook/WebhookSimpleSmsSender.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import com.sflpro.notifier.spi.sms.SimpleSmsSender;
+
+/**
+ * Created by Hayk Mkrtchyan.
+ * Date: 6/24/19
+ * Time: 11:46 AM
+ */
+class WebhookSimpleSmsSender extends AbstractWebhookSmsSender<SimpleSmsMessage> implements SimpleSmsSender {
+
+    public WebhookSimpleSmsSender(final WebhookApiRestClient apiRestClient) {
```
Comment (mkrtchyanh) [inline infra/infra-integrations/infra-integrations-sms/src/main/java/com/sflpro/notifier/externalclients/sms/webhook/WebhookSimpleSmsSender.java:14]: Maybe package access is better?

**Review by mkrtchyanh — COMMENTED** [review]
God job!

File: infra/infra-integrations/infra-integrations-sms/src/main/java/com/sflpro/notifier/externalclients/sms/webhook/WebhookSimpleSmsSender.java (hunk 1 above)
Comment (nazaretyan) [reply infra/infra-integrations/infra-integrations-sms/src/main/java/com/sflpro/notifier/externalclients/sms/webhook/WebhookSimpleSmsSender.java:14]: Agree!! Will change to package access

File: infra/infra-integrations/infra-integrations-sms/pom.xml [hunk 2]
```diff
@@ -70,5 +70,9 @@
             <groupId>com.fasterxml.jackson.module</groupId>
             <artifactId>jackson-module-jaxb-annotations</artifactId>
         </dependency>
+        <dependency>
```
Comment (mikron) [inline infra/infra-integrations/infra-integrations-sms/pom.xml:73]: Do we need this dependency now?

**Review by mikron — CHANGES_REQUESTED** [review]

File: infra/infra-integrations/infra-integrations-sms/pom.xml (hunk 2 above)
Comment (nazaretyan) [reply infra/infra-integrations/infra-integrations-sms/pom.xml]: @mikron jan removed

**Review by mikron — APPROVED** [review]

**Comment by dnavre** [comment]
testing travis integration

**Pull request merged**